src/lib/viewers/image/ImageViewer.js

```diff
@@ -37,7 +37,9 @@ class ImageViewer extends ImageBaseViewer {
         this.updatePannability = this.updatePannability.bind(this);
 
         this.annotationControlsFSM = new AnnotationControlsFSM(
-            this.options.enableAnnotationsImageDiscoverability ? AnnotationState.REGION_TEMP : AnnotationState.NONE,
```

Comment:
It seems like we would need to update all usages of `this.options. enableAnnotationsImageDiscoverability` to use this new method?

Reply:
good point 👍